spec/requests/verify_authentications_spec.rb

```diff
@@ -129,4 +129,14 @@
       expect(response).to redirect_to(root_path)
     end
   end
+
+  def generate_claim!
```

Comment:
is this a duplicate of `create_claim` that's defined in `RequestHelpers`?

Reply:
Whoops, yeah, I forgot to take that out